Split/green belt request

Computer8543 opened the pull request:
Belt Reflection:

In the future as a black belt, I will be a leader who not only acts but acts effortlessly. I do not desire to command, but to break change impedance through direct action and to inspire others to follow me. I will be able to focus more on people and relationships than processes when acting and reflecting. That will allow me to build a base of support, a movement, to enact the change that is needed and to thrive in an ever-changing environment. When I become that leader, the exact title I will hold will be irrelevant since the representation of a thing is not the same as the actual thing. What will be relevant is my soft and hard skills, which will enable me to be the person I desire to be. 

What I have learned so far overall and specifically at the belt level is that I had to change my perception of the relationships around me and their potential. For example, my perception of my relationship with Michael. When I internalized that he was influencing me, I first got very angry at him and myself. I felt that I was losing the rhetorical battle due to my emotional immaturity despite being able to keep up with him intellectually. Then I realized to effectively change mindsets, to be a true Mindset Sensei, one must be indirect. They must lead people’s thoughts in the direction that they desire so that the students think they have those thoughts themselves and thus internalize their teachings. Furthermore, I also realized that my frame of reference was wrong, that seeing this as a battle or a zero-sum game, instead of a dance or a positive sum game, ended up limiting how far I could grow. Thus, I choose the middle path of engaging with him in a more careful and balanced way to achieve mutually beneficial goals.

Another thing I learned at this belt level, is that I add too much, and do not subtract enough. The first step to act effortlessly is to remove all unnecessary action, and the second is to repeat the necessary actions over and over again until they become a reflex. It is not enough to simply act effortlessly, one must also reflect effortlessly to integrate their insights. 

I intend to apply what I have learned in the short and medium term by negotiating the growth of my unpaid IT internship into a part-time IT job. To get a good deal, I will need to negotiate well and wisely. I feel that I will be prepared for that when the time comes. Before that, this summer, I plan on leading an effort to clean and organize my family’s house. Cleaning my environment, and making habits to keep it clean and organized, will change my mental state to be more organized and effective. That will affect everything I do, and further enable me to progress in the Negotiation Mastery Program.


Green Belt Challenge:

Link to Spirited Conversation:
https://www.youtube.com/watch?v=f1C6b2Wd8HM

FEBI Energy Analysis:
The coach has powerful Driver-Visionary Energy, since he is not only anticipating potential failure, but inspiring the team to create a better outcome. He is inviting the team to a model of success based on shared values, so some Collaborator-Organizer energy is at play. There is also a change in focus in the FEBI energies during the speech, from Visionary in the beginning to Collaborator-Organizer at the end, with a consistent throughline of Driver energy. This imbalance in energies is enabling, since the Visionary energy at the beginning sets up the likely negative result as well as the solution to that problem, the Driver energy moves things forward, and the Collaborator-Organizer energy at the end creates a consensus around shared values and strategic vision.  The coach successfully unites the team behind a shared vision of success in no small part due to the synergy of all four FEBI energies. 

Cynefin Model Analysis:
The coach makes a disordered situation simple, avoiding potential chaos, through the mantra “One more inch”. All the rules and complexities of football, distilled into their essence into a principle that not only defines the sport but life itself. Football itself is a complicated situation because there are strict rules, an easily measurable outcome, and an existing metagame. There are still degrees of freedom and thus influence that the players have within the game. However, the team is in a disordered situation before the coach’s speech because of their previous failures and lack of shared strategic vision. The team is very close to internal chaos and falling apart due to potentially losing their last chance to succeed, but the coach unites them around a shared ideal and avoids that fate.

Map of Consciousness Analysis:
The team starts in Guilt and/or Shame, and through the coach’s appeal to their shared values and distillation of a core principle, combining into a shared vision of success, I would say that the team ends at least in Courage, with shades of Enlightenment.

abkaufmann commented:
This feels like a more substantial article. I wasn’t a huge fan of the section that referred to the military, but it is short. The examples provided are more helpful than in the previous article.

abkaufmann reviewed (approved):
Approving change - more substantial article.

Computer8543 committed:
Initial draft of Negotiation Mastery Program in a YAML file format.

Computer8543 committed:
Merge pull request #1 from Computer8543/kyle

Merge kyle branch into main branch

Computer8543 committed:
Merge Initial Draft code from main branch to begin negotiation. Also removed comments to make change so I can git commit directly to this issue.

Computer8543 committed:
Merge pull request #2 from Computer8543/main

Merge main branch changes into kyle branch

Computer8543 committed:
Merge branch 'mindset-dojo:main' into kyle

Computer8543 committed:
Create LICENSE.md

Add CC BY-SA 4.0 license to repo.

Computer8543 committed:
Create TODO.md

Create an initial draft of a minimal set of contribution guidelines for the Mindset Dojo repo.

Computer8543 committed:
Create README.md

Created README.md file to summarize the offerings of Mindset Dojo and it's community succinctly.

Computer8543 committed:
Merge pull request #3 from Computer8543/kyle

Merge kyle branch into main branch

Computer8543 committed:
Rename TODO.md to CONTRIBUTING.md

This is to follow open source naming conventions.

Computer8543 committed:
Merge pull request #4 from Computer8543/kyle

Rename TODO.md to CONTRIBUTING.md

Computer8543 committed:
Green Belt Push Request and Tag Change

Computer8543 committed:
Merge pull request #5 from Computer8543/kyle

Merge kyle branch into main

Computer8543 committed:
Delete README.md

Computer8543 committed:
Delete LICENSE.md

Computer8543 committed:
Delete CONTRIBUTING.md

Computer8543 committed:
Delete _data/negotiation-mastery-program.yaml

michael-basil reviewed (approved):
These are fantastic reflections.

I will tentatively approve them pending our discussion tomorrow.

I have tagged @aphor as well to take a look.

Please feel free to share on the Mat!
